docs/developer-docs/latest/developer-resources/content-api/integrations/go.md

```diff
@@ -0,0 +1,319 @@
+# Getting Started with GO
+
+This integration guide is following the [Getting started guide](/developer-docs/latest/getting-started/quick-start.md). We assume that you have completed [Step 9](/developer-docs/latest/getting-started/quick-start.md#_9-consume-the-content-type-s-api) and therefore can consume the API by browsing this [url](http://localhost:1337/restaurants).
```

Comment:
```suggestion
This integration guide is following the [Getting started guide](/developer-docs/latest/getting-started/quick-start.md). We assume that you have completed [Step 8](/developer-docs/latest/getting-started/quick-start.md#_8-publish-the-content) and therefore can consume the API by browsing this [url](http://localhost:1337/restaurants).
```

Reply:
(we [reverted the previous link](https://github.com/strapi/documentation/pull/191/files) on this line because it makes more sense)